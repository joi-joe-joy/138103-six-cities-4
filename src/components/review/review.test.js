import React from "react";
import renderer from "react-test-renderer";
import Review from "./review.jsx";

const review = {
  id: 5,
  author: {
    photo: `img/avatar-angelina.jpg`,
    name: `Monica`
  },
  description: `bedrooms â€“ couples and groups of friends will find this accommodation`,
  date: 1593095836000,
  rating: 3
};


describe(`Render Review`, () => {
  it(`Render Review correctly`, () => {
    const tree = renderer
      .create(<Review review={review}/>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});

