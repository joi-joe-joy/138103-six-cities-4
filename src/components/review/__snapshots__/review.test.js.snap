// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Review Render Review correctly 1`] = `
Array [
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Monica"
        className="reviews__avatar user__avatar"
        height="54"
        src="img/avatar-angelina.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Monica
    </span>
  </div>,
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "60%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      bedrooms â€“ couples and groups of friends will find this accommodation
    </p>
    <time
      className="reviews__time"
      dateTime="2020-06-25"
    >
      June 2020
    </time>
  </div>,
]
`;
